### DISCUSSION TITLE
Blog: is it possible to associate a URL with the blog author?

### DISCUSSION THREAD

**Discussion by aborruso** [squidfunk/mkdocs-material#5953] [Q&A]
Hi,
is there a way to associate a URL to blog authors, and have the hyperlink in logo and/or name?

Thank you

![image](https://github.com/squidfunk/mkdocs-material/assets/30607/06d841ce-5bce-44ec-b408-5f2e89e88375)

**Comment by squidfunk** [comment]
Not yet, but you can open a feature request if you like, explaining how you would expect it to work. We're thinking about generating author indexes where authors can create a profile for them, and all blog posts of the author are linked.

**Reply by aborruso** [reply]
@thank you @squidfunk. This is the feature request https://github.com/squidfunk/mkdocs-material/issues/5955
